core/spec/models/spree/order_cancellations_spec.rb

```diff
@@ -152,7 +152,9 @@
 
       it "generates the correct total amount" do
         order.cancellations.short_ship([inventory_unit_1])
+        inventory_unit_2.line_item.reload # UnitCancel#compute_amount needs updated amounts
         order.cancellations.short_ship([inventory_unit_2])
+        line_item.reload
```

Comment:
This and the change above it are unfortunate. :(  See the [commit message](https://github.com/solidusio/solidus/pull/1389/commits/164971be3e4c2ac9dc14606af532c6e0408a3345) for details.